boilerplate/app/components/Screen.tsx

```diff
@@ -197,18 +205,15 @@ export function Screen(props: ScreenProps) {
     KeyboardAvoidingViewProps,
     keyboardOffset = 0,
     safeAreaEdges,
-    SafeAreaViewProps,
     StatusBarProps,
     statusBarStyle = "dark",
   } = props
 
-  const Wrapper = safeAreaEdges?.length ? SafeAreaView : View
+  const insetPadding = useSafeAreaInsetPadding(safeAreaEdges)
 
   return (
-    <Wrapper
-      edges={safeAreaEdges}
-      {...SafeAreaViewProps}
-      style={[$safeAreaStyle, SafeAreaViewProps?.style, { backgroundColor }]}
+    <View
+      style={[$containerStyle, { backgroundColor, ...insetPadding }]}
```

Comment:
Probably prefer this style.

```suggestion
      style={[$containerStyle, { backgroundColor }, insetPadding ]}
```